Library/Formula/aap.rb

```diff
@@ -0,0 +1,22 @@
+class Aap < Formula
+  desc "Aap is a make-like tool to download, build, and install software"
+  homepage "http://www.a-a-p.org"
+  url "https://downloads.sourceforge.net/project/a-a-p/Aap/1.093/aap-1.093.zip"
+  sha256 "7a6c6c4a819a8379e60c679fe0c3f93eb1b74204cd7cc1c158263f4b34943001"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    # Aap installs "man" at top level. This moves it under share/.
+    inreplace "main.aap", "mandir = $PREFIX/man/man1", "mandir = $PREFIX/share/man/man1"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!